packages/kbn-pm/src/utils/projects.ts

```diff
@@ -48,6 +51,10 @@ export async function getProjects(
       const projectDir = path.dirname(projectConfigPath);
       const project = await Project.fromPath(projectDir);
 
+      if (workspaceProjectsPaths.indexOf(filePath) >= 0) {
+        project.isWorkspaceProject = true;
```

Comment:
Not a big fan of setting this here. Would rather that the `Project.fromPath` function set this up (would need to pass in the rootDir/root package.json info)